azure-communication-ui/azure-communication-ui-demo-app/src/main/java/com/azure/android/communication/ui/callingcompositedemoapp/features/AdditionalFeatures.kt

```diff
@@ -0,0 +1,44 @@
+package com.azure.android.communication.ui.callingcompositedemoapp.features
+
+import android.content.Context
+import android.os.Build
+import com.azure.android.communication.ui.callingcompositedemoapp.R
+import com.azure.android.communication.ui.callingcompositedemoapp.diagnostics.FpsDiagnostics
+import com.azure.android.communication.ui.callingcompositedemoapp.diagnostics.MemoryViewer
+import com.azure.android.communication.ui.utilities.FeatureFlagEntry
+import com.azure.android.communication.ui.utilities.FeatureFlags
+
+// / Extra rules for enabling the Diagnostics feature
```

Comment:
Code is very clear :) I think place where we are using comments, we can follow // .